src/integrationtest/resources/templates/instances.yml

```diff
@@ -108,3 +126,5 @@ Resources:
                 b: !Ref someParameterB
                 c: "foo"
         kms_encrypted_value: "|Ref|kmsEncryptedValue"
+        ssm_plain_value: "|Ref|ssmPlainValue"
```

Comment:
What's going on here?

Reply:
It's testing the SSM keyword as far as I can see?